internal/gravatar/gravatar.go

```diff
@@ -1 +1,11 @@
 package gravatar
+
+import (
+	"crypto/md5"
+	"strings"
+)
+
+func GetURL(email string) string {
+	r := md5.Sum([]byte(strings.TrimSpace(strings.ToLower(email))))
+	return string(r[:])
+}
```

Comment:
The name suggests it should return an URL but it seems that it returns only md5 hash